deployment/README.md

```diff
@@ -0,0 +1,71 @@
+
+#### User Data
+
+You will upload your own DistrictBuilder `config.xml` and shapefile zip to the App Server as a part of provisioning. Simply add a DistrictBuilder `config.xml` file and `districtbuilder_data.zip` in the [`user-data`](./user-data/) folder, and the provisioner will upload them to the server for you. 
+
+#### `scripts/infra`
+Once the settings bucket and User Data are configured, you can run the deployment. To deploy the infrastructure for a DistrictBuilder instance (EC2, Route53 records, etc.) resources, use the `infra` wrapper script to lookup the remote state of the infrastructure and assemble a plan for work to be done. You can run `scripts/infra` inside of the `terraform` Docker container defined in `docker-compose.ci.yml`, which contains all of the necessary software dependencies for deployments.
+
+First, obtain your account's AWS API keypair, and add them to your environment. Then, set `DB_SETTINGS_BUCKET` and `IMAGE_VERSION`, and run `scripts/infra`:
+
+
+```bash
+$ export AWS_ACCESS_KEY_ID="****************F2DQ"
+$ export AWS_ACCESS_KEY_ID="****************TLJ/"
+$ export DB_SETTINGS_BUCKET="districtbuilder-staging-config-us-east-1"
+# IMAGE_VERSION can be a git SHA, or version tag
+$ export IMAGE_VERSION=123456"
+$ docker-compose -f docker-compose.ci.yml run --rm terraform ./scripts/infra plan
+```
+
+Once the plan has been assembled, and you agree with the changes, apply it:
+
+```bash
+$ docker-compose -f docker-compose.ci.yml run --rm terraform ./scripts/infra apply
+```
+
+This will attempt to apply the infrastructure plan assembled in the previous step using Amazon's APIs and use `docker-compose` to configure the App Server with User Data, build container images, start services and run migrations. In order to change specific attributes of the infrastructure, inspect the contents of the environment's `tfvars` file in Amazon S3. To modify application variables, change `.env`.
+
+#### Loading Shapefile data
+To load data, configure the deployment machine to communicate with the Docker Daemon on the App server, then run `./scripts/load_configured_data --production`.
+
+```bash
+# Download docker client certificates from S3 and install them
+$ aws --profile district-builder s3 cp s3://${DB_SETTINGS_BUCKET}/docker_certs/client/client.zip
+$ unzip -d /path/to/docker/certs client.zip
+# Enable Docker over TLS
+
+$ export DOCKER_HOST=origin.${DB_STATE}.districtbuilder.azavea.com:2476
+$ export DOCKER_TLS_VERIFY=1
+$ export DOCKER_CERT_PATH=/path/to/docker/certs
+$ ./scripts/load_configured_data --production
```

Comment:
Should this be updated with the additional instructions you gave me?

Reply:
Yup, I'll update that now.